agent/engine/docker_container_engine_test.go

```diff
@@ -409,10 +409,11 @@ func TestStopContainerTimeout(t *testing.T) {
 	mockDocker, client, testTime, done := dockerclientSetup(t)
 	defer done()
 
+	dockerStopTimeout := client.config.DockerStopTimeoutSeconds
```

Comment:
I think you want this to be `client.config.DockerStopTimeoutSeconds + stopContainerAPITimeout`.